test/libsolidity/SMTCheckerTest.cpp

```diff
@@ -136,6 +136,9 @@ SMTCheckerTest::SMTCheckerTest(std::string const& _filename):
 
 	auto const& bmcLoopIterations = m_reader.sizetSetting("BMCLoopIterations", 1);
 	m_modelCheckerSettings.bmcLoopIterations = std::optional<unsigned>{bmcLoopIterations};
+
+	if (m_evmVersion < langutil::EVMVersion::prague() && CommonOptions::get().eofVersion().has_value())
+		m_shouldRun = false;
```

Comment:
The `prague` check is redundant.
```suggestion
	if (CommonOptions::get().eofVersion().has_value())
		m_shouldRun = false;
```